the contbuild caught an issue:

```
AssertionError: TypeError('unbound method _lazy_new() must be called with DoubleTensor instance as first argument (got type instance instead)',) is not an instance of <type 'exceptions.RuntimeError'>
```

--- a/torch/cuda/__init__.py
+++ b/torch/cuda/__init__.py
@@ -261,6 +261,7 @@ if not hasattr(torch._C, 'CudaDoubleStorageBase'):
     torch._C.__dict__['_CudaStreamBase'] = _dummy_type('CudaStreamBase')
 
 
+@staticmethod
 def _lazy_new(cls, *args, **kwargs):
     _lazy_init()
     # We need this method only for lazy init, so we can remove it

